sceptre/resolvers/stack_output.py

```diff
@@ -163,21 +173,43 @@ def resolve(self):
         """
         self.logger.debug("Resolving external Stack output: {0}".format(self.argument))
 
-        profile = None
-        region = None
-        sceptre_role = None
         arguments = shlex.split(self.argument)
```

Comment:
It doesn't appear as if you're still need `arguments`.

Reply:
Yes, it's still needed.